src/status_im/ui/components/chat_icon/screen.cljs

```diff
@@ -8,8 +8,6 @@
             [status-im.ui.screens.chat.photos :as photos]
             [status-im.utils.platform :as platform]))
 
-;;TODO REWORK THIS NAMESPACE
```

Comment:
this still need to rework it :)

Reply:
oh sorry, that was removed by mistake, generally all caps comment are the things I leave before commiting :)